server/routes/api/collections.ts

```diff
@@ -148,6 +148,21 @@ router.post("collections.info", auth(), async (ctx: APIContext) => {
   };
 });
 
+router.post("collections.documents", auth(), async (ctx: APIContext) => {
```

Comment:
New endpoint for getting the collection structure, it will no longer be returned from `collections.info` and requires the "readDocument" permission which admins do not have by default.